client/viewer/js/main.js

```diff
@@ -36,6 +36,22 @@ socket.on('update', function(data){
 	viewer.update(data);
 });
 
+socket.on('reassign', function(data){
+
+	console.log("Reassign", data);
```

Comment:
Can we use the debug module with the namespace beginning with `screens:`. The reason being `console.log` is a blocking call and always logs to `stdout`, we may only want to listen to logs from a specific module.